### DISCUSSION TITLE
Gulp command not working

### DISCUSSION THREAD

**Discussion by JOY2OP** [gulpjs/gulp#2713] [Help]
I'm using the volt bootstrap dashboard, and whenever I try to run the gulp command instead of running the server it shows this error `[15:34:55] Finished 'scss' after 4.42 s
[15:34:55] Starting 'html'...
[15:34:55] 'html' errored after 69 ms
[15:34:55] Error in plugin "gulp-file-include"
Message:
    JSON5: invalid character '\r' at 1:43
Details:
    domainEmitter: [object Object]
    domainThrown: false

[15:34:55] 'serve' errored after 4.51 s
[15:34:55] 'default' errored after 4.52 s`

**Comment by milenairon** [comment]
Там где подключаешь @@include('html/ExSlider.html', {}) не делай перенос строки. У меня и-за этого такая ошибка вылетала.
